sympy/printing/str.py

```diff
@@ -1007,6 +1017,85 @@ def _print_Str(self, s):
         # Str does not to be printed same as str here
         return "%s(%s)" % (s.__class__.__name__, self._print(s.name))
 
+# class StrDecPrinter(StrPrinter):
```

Comment:
Do not add commented out code like this. If the code is not ready yet, you can discuss it on the issue, but it shouldn't be added like this.